src/test/emwasm_compile_known_gcc_test_failures.txt

```diff
@@ -5,33 +5,30 @@
 	20010605-1.c
 	20020412-1.c
 	20030501-1.c
-	20040302-1.c
 	20040308-1.c
 	20040423-1.c
 	20040520-1.c
 	20041218-2.c
 	20061220-1.c
 	20070919-1.c
-	20071210-1.c
+  20071220-1.c O3
```

Comment:
Oh no I tried to fix the indentation errors by 'resolve conflicts' tool, which didn't show me tabs, and now they are even more unaligned no